frontend/pages/ManageControlsPage/OSSettings/cards/CustomSettings/components/ProfileListItem/ProfileListItem.tsx

```diff
@@ -89,7 +89,7 @@ const ProfileListItem = ({
   const onClickDownload = async () => {
     const fileContent = await createFileContent(profile);
     const formatDate = format(new Date(), "yyyy-MM-dd");
-    const extension = createProfileExtension(profile);
+    const extension = createProfileExtension(profile); // TODO: How does createProfileExtension work with iOS/iPadOS profiles
```

Comment:
Indeed. Do we need to answer this before merging?

Reply:
Yes, or we can merge and then ask